setup.cfg

```diff
@@ -39,6 +39,9 @@ install_requires =
     astor
     attrs
 
+[options.extras_require]
+docs = Sphinx; sphinx_rtd_theme
```

Comment:
I think that the comment should be removed and add `sphinx_rtd_theme`

In this way, we can run  `pip install -e [.docs]` and be able to re-create the docs on local dev system